infra/base-images/base-runner/test_all

```diff
@@ -38,6 +38,10 @@ for FUZZER_BINARY in $(find $OUT/ -maxdepth 1 -executable -type f); do
     continue
   fi
 
+  if grep -va $FUZZER_BINARY "LLVMFuzzerTestOneInput" > /dev/null 2>&1; then
```

Comment:
Can you also add that check in base-images/base-runner/coverage and in base-images/base-runner/test_report (is it used anywhere, btw?)

Reply:
Sorry for the confusing comment. I meant "is base-images/base-runner/test_report used anywhere?"

Re ClusterFuzz: yeah, I remember

Reply:
Ah no idea about this. I can look

Reply:
The purpose of the file is:
```
# Test fuzzers and writes junit xml reports.
```
I suspect it isn't used. Will update but open a new bug removing it in case I'm right.